unit/Makefile

```diff
@@ -9,6 +9,7 @@ SRC = unit_tests.cpp \
 SRC += unit_tests.cpp \
        analyses/ai/ai_simplify_lhs.cpp \
        analyses/call_graph.cpp \
+       analyses/graph.cpp \
```

Comment:
Preferably this would have a name that coincides with the class this is testing. Now you are testing `grapht` functionality using code from `analyses/` - maybe you can come up with some name that states in a less ambiguous way what is being tested?